app/config/FrontendAppConfig.scala

```diff
@@ -33,6 +33,7 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   lazy val reportAProblemNonJSUrl = s"$contactHost/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
   lazy val betaFeedbackUrl = s"$contactHost/contact/beta-feedback"
   lazy val googleTagManagerId: String = configuration.get[String](s"google-tag-manager.id")
+  lazy val optimizelyId: String = configuration.get[String](s"optimizely.projectId").concat(".js")
```

Comment:
Does this need the 'concat' at the end?

Reply:
figured how to add the ".js" in twirl without messing things up.